indico/MaKaC/webinterface/rh/categoryDisplay.py

```diff
@@ -61,6 +61,9 @@ class RHCategDisplayBase(base.RHDisplayBaseProtected):
     def _checkParams(self, params, mustExist=True):
         if "categId" in params:
             params["categId"] = escape_html(str(params["categId"]))
+        # TODO: Workaround to avoid breaking the category navigation (display)
```

Comment:
Well, I put it there for a reason, why get rid of it?
I suppose this is not the optimal way to do it, but at some point in the future when we rewrite the event creation page to only use WTForms, we'll have to change this as well. I could remove the TODO and leave the comment, but I don't mind it.